content/45.user_friendliness.md

```diff
@@ -1,7 +1,67 @@
 ## User-Friendliness
 
+Several different entrypoints exist for research codes.
+For instance, an ontology of "end-users" in science has developed that emphasizes that typically "library" codes are only used implicitly by researchers.
+In this framing, library codes primarily interface with application codes, rather than with researchers.
+We have taken a slightly different approach with the design and development of yt, as it exists in a middle-ground as a library used *as an application* within the scripting language python and also as a library for more complex analysis of data.
```

Comment:
```suggestion
We have taken a slightly different approach with the design and development of yt, as it exists in a middle-ground as a library used *as an application* within the scripting language Python and also as a library for more complex analysis of data.
```

what part of yt would you describe as an application ? I would think of the CLI as such, but is there anything else ?

Reply:
So, my phrasing was kind of deliberate -- it's a library that people use as an application.  Basically, a lot of the stuff they do is what amounts to GUI operations mapped onto a reasonably small API.  I'm not saying it's an application, but I would argue that it's used similarly to one.

Reply:
I see, then I have no issues with that